### DISCUSSION TITLE
req.created_at Bugs

### DISCUSSION THREAD

**Discussion by Rhynorater** [caido/caido#1618] [Q&A]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Bug description

It doesn't appear that `req.created_at` is working. Perhaps this is only the case with older projects?

https://www.loom.com/share/ea30f7e028494f2ea9fdef83aaca0798

It does seem to work with newer projects. However, adding a time does **not** work. 

https://www.loom.com/share/fb124bb3a9264f51a3a65c1eb7123d92

### How to reproduce

1. Boot up caido and make some requests
2. Attempt to apply the filters

### Environment & setup

OS: Windows
OS version: 11
Caido client: Caido Desktop
Caido version: 0.38

**Comment by Sytten** [comment]
Alright I banged my head all morning to figure that out.

- First bug: You entered `req.created_at.gt:"2024-06-31"`. There is no 31st day in June, thats why you didn't have rows. This is a UI problem, we will add a time picker. Classifying that as "work as intented"

- Second bug: This is a legit bug due to the lack of timezone. Again it will be mitigated with a UI time picker. Essentially the column is displayed in your browser timezone but the HTTPQL query uses UTC. To fix that use a timezone so `req.created_at.gt:"2024-07-04T23:10:00-07:00"`

**Comment by Sytten** [comment]
Closing in favour https://github.com/caido/caido/issues/1119
